### DISCUSSION TITLE
One file pack ussue

### DISCUSSION THREAD

**Discussion by Hbver** [dashingsoft/pyarmor#1629] [Q&A]
Hello. i need to make a obf exe file from py. i had a problem with second step in the docs. where am i wrong?

![Screenshot_2](https://github.com/dashingsoft/pyarmor/assets/153429778/ae040b2b-a282-4721-a996-401b3ad3b311)

C:\OBF>pyarmor gen -O obfdist --pack dist/filename filename.py
INFO     Python 3.12.0
INFO     Pyarmor 8.4.4 (trial), 000000, non-profits
INFO     Platform windows.x86_64
INFO     implicitly set output to ".pyarmor\pack\dist"
ERROR    --pack must be an executable file

**Comment by HangingWithCMM** [comment]
Try using “pyarmor-webui” it makes everything easier, especially this.

**Comment by Deeka0** [comment]
Since you are running Windows, I suggest you are to pass `--pack dist/filename.exe` rather than `--pack dist/filename`.
So it becomes: `C:\OBF>pyarmor gen -O obfdist --pack dist/filename.exe filename.py`
